app/controllers/request_controller.rb

```diff
@@ -939,4 +935,13 @@ def handle_blocked_ip(info_request)
     end
   end
 
+  def set_info_request
+    AlaveteliLocalization.with_locale(@locale) do
+      @info_request ||= InfoRequest.find_by_url_title!(params[:url_title])
+    end
+  end
+
+  def set_locale
+    @locale ||= AlaveteliLocalization.locale
```

Comment:
Not for now but it would be nice is this method was just called `locale` and made a helper method so we don't need to use the `@locale` instance var everywhere. Would mean the order the of the callbacks wouldn't matter too.

Reply:
Agreed – fixed, albeit local to this controller. Can't see why we don't move this to `ApplicationController`, but not doing that now.